src/plot.jl

```diff
@@ -179,8 +179,8 @@ function Plot(
 ) where {C<:Canvas}
     length(xlim) == length(ylim) == 2 ||
         throw(ArgumentError("xlim and ylim must be tuples or vectors of length 2"))
-    length(X) == length(Y) || throw(DimensionMismatch("X and Y must be the same length"))
-    width = max(width, min_width)
+    length(X) == length(Y) || throw(DimensionMismatch("X and Y must have same length"))
+    (visible = width > 0) && (width = max(width, min_width))
```

Comment:
It's perhaps a matter of taste here: this version might let people wondering if `width` is still used when `visible=false`. For instance, does the following version work the same?

```julia
visible = width > 0
width = max(width, min_width)
```

Reply:
I find it a bit counterintuitive to have `visible == false` when `width = 0` and then force a non-zero width afterwards with `max(...)`. Anyways I've checked and both versions pass our tests.

Reply:
> I find it a bit counterintuitive to have visible == false when width = 0 and then force a non-zero width afterwards with max(...)

Makes sense. I was thinking that we can assume `width` is not used whenever `visible==false` for entire UnicodePlots; not just this function.

Reply:
We could also have added a `canvas_visible` kwarg to `Plot(...)`, but I though it was a bit of an overkill here.